pkg/csi/service/common/util.go

```diff
@@ -184,7 +184,7 @@ func validateVolumeCapabilities(volCaps []*csi.VolumeCapability,
 			// ext3, ext4, xfs for Linux and ntfs for Windows.
 			if volCap.GetMount() != nil && !(volCap.GetMount().FsType == Ext4FsType ||
```

Comment:
would we have the same problem with these other fsTypes?

Reply:
Maybe. I haven't had time to evaluate all the Linux filesystems to see if this is also a problem there.